stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>Jennifer Ding:</p><ul><li>There are other data shorthands that may be confusing to a domain outsider, like myself. For example, the types of glass. What is their significance, e.g Vehicle Float vs. Non-Float?</li></ul><p>Joe Cerniglia:</p><p>I agree there is a real need here for explanations of the types of glass.&nbsp;I will provide that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>